lib/datadog/core/telemetry/collector.rb

```diff
@@ -179,17 +177,15 @@ def products
         end
 
         def profiler
-          version = profiler_version
-          Telemetry::V1::Profiler.new(version: version) if version
+          { version: profiler_version } if profiler_version
         end
```

Comment:
Since we capture `profiling.enabled` in the `additional_payload`, I will change this to always send the profiler version in a future PR (and do the same for appsec)